ttnn/cpp/ttnn/config.hpp

```diff
@@ -90,6 +91,18 @@ struct Config {
             }
         }
 
+        if (
+            name == "enable_detailed_tensor_report_pytorch_converted") {
+            if (not this->attributes.enable_detailed_tensor_report) {
+                if (this->attributes.enable_logging) {
+                    tt::log_warning(
+                        tt::LogAlways,
+                        "Running without detailed tensor report. Please enable detailed tensor report mode if you want "
+                        "to enable saving all tensors as pytorch.");
```

Comment:
I think we should rephrase. 
"to enable saving all tensors as pytorch" is talking about what the flag does technically, but not what it enables.

Something like:
"Enable detailed tensor report by [DOING X] to enable [WHAT USER WILL BE ABLE TO DO WITH IT]"

Reply:
Changed the wording here. If you have a specific phrasing you'd like let me know and I'll make that change.